test/runtime/samples/style_manager-cleanup/_config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+	skip_if_ssr: true,
+	skip_if_hydrate: true,
+	skip_if_hydrate_from_ssr: true,
+	test({ raf, assert, component, window }) {
+		component.visible = true;
+		raf.tick(100);
+		component.visible = false;
+		raf.tick(200);
+		raf.tick(0);
```

Comment:
```suggestion
		// calling raf again as the cleanup happens in the next raf callback
		raf.tick(200);
```

Reply:
@MathiasWP probably can comment to explain why the extra call of `raf.tick()` is necessary for the test

Reply:
@tanhauhau I am having some issues with running the test, and i don't know why. This test fails on my computer, and I am wondering if i set the extra `raf.tick()` to make it work initially.

But now the test fails every time, and I cannot understand why. Is there some documentation about how the tests in Svelte work, so that i can try and figure out why this happens?